plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -163,6 +163,7 @@ public open class HtmlRenderer(
                     contentTabs.forEachIndexed { index, contentTab ->
                         button(classes = "section-tab") {
                             if (index == 0) attributes["data-active"] = ""
+                            attributes["data-type"] = pageContext.getDocumentableType() ?: ""
```

Comment:
Oh btw, it might be worth adding a test that verifies that this attribute exists on the pages, and adding a comment to these tests that the value is used in js tab-handling scripts wouldn't hurt. Should be a simple and quick intro into writing html tests as well

We don't have any UI tests, so if this gets lost during refactoring or someone thinks it's not needed, it might lead to regression.

Reply:
@vmishenev thanks! You are right, I was thinking, that it will be harder to reusing it, but in the end it simplified code a lot :)
Pushed changes with usage of it.

So I will now add test for checking that just `data-page-type` attribute is present on page